phpBB/phpbb/language/exception/invalid_plural_rule_exception.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\language\exception;
+
+/**
+ * Exception thrown by \phpbb\language\language::get_plural_form()
+ */
+class invalid_plural_rule_exception extends \phpbb\exception\runtime_exception
```

Comment:
(a base class or an interface) the idea being to be able to catch every language related exceotion and not the others.